### PR TITLE
Update entries for Beta and Certified Packages

### PR DESCRIPTION
**Pull request by vishnu2kmohan**
## Description
Update entries for Beta and Certified Packages

Note: This PR includes the fix for the merge conflict from https://github.com/mesosphere/dcos-docs-site/pull/1846 which can be closed in favor of this PR
## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/version-policy/index.md [hunk 1]
```diff
@@ -39,17 +38,14 @@ Thus, upon the release of 2.0.0 where 1.10.4 and 1.9.7 are available:
 * **Minor Version**: Upon the availability of a new DC/OS Minor version 1.11.0, minor versions prior to and including 1.8.0 will no longer be supported as (N-2) includes 1.11 (N), 1.10 (N-1) and 1.9 (N-2).
 * **Maintenance Version**: Upon the availability of DC/OS Maintenance version 1.10.5, Maintenance versions prior to and including 1.10.0 will no longer be supported as (N-4) includes 1.10.5 (N), 1.10.4 (N-1), 1.10.3 (N-2), 1.10.2 (N-3) and 1.10.1 (N-4).<br>
 
-
 ## DC/OS Platform Version Compatibility Matrix
 
 The following matrix shows the platform components and operating environments on which DC/OS has been tested; DC/OS will run on these components. Customers running DC/OS on non-supported platform components should upgrade to a supported component. For clarity, Mesosphere only provides support services to paying customers under a written agreement; the term “supported” in Mesosphere documentation refers to whether the indicated software component has been tested for compatibility.
```
Comment (chrisgaun) [inline pages/version-policy/index.md]: We should include warnings about the fact that if customers are using a RHEL or CentOS then Mesosphere Kubernetes Engine requires CentOS/RHEL 7.5+ due to a well known kernel bug in that affects Kubernetes on all older operating systems. I am having trouble finding in the docs where we have put this warning before.

File: pages/version-policy/index.md [hunk 2]
```diff
@@ -244,358 +234,242 @@ The designations are as follows:
...
+        <td>Kafka-Zookeeper 2.4.x-3.4.13 (Recommended)</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
-        <td><p style="text-align: center;">◯</p></td>
     </tr>
     <tr>
-        <td>Kubernetes 1.0.x-*</td>
+        <td>Kubernetes 1.3.x-1.10.y</td>
         <td><p style="text-align: center;">⚫</p></td>
```
Comment (chrisgaun) [inline pages/version-policy/index.md]: Remove the one about these packages being supported on DC/OS 1.12. We have tested them for the purpose of migration but not for long term use.

File: pages/version-policy/index.md (hunk 1 above)
Comment (vishnu2kmohan) [reply pages/version-policy/index.md]: This warning already exists in the fully rendered page.

File: pages/version-policy/index.md (hunk 2 above)
Comment (vishnu2kmohan) [reply pages/version-policy/index.md]: Done

**Comment by masonse** [comment]
What do the "*" (asterisks) mean in the table?
For the browser compatibility table, please center the symbols.

**Comment by razi-malik** [comment]
Just had one comment regarding 1.2.0 version. Otherwise looks good.

**Comment by razi-malik** [comment]
> What do the "*" (asterisks) mean in the table?
> For the browser compatibility table, please center the symbols.

It means its supported for that version.

**Comment by SarahStegall** [comment]
Looks good to me.

**Pull request merged**